src/Framework/LazyFormattedBuildEventArgs.cs

```diff
@@ -179,6 +180,8 @@ internal override void CreateFromStream(BinaryReader reader, Int32 version)
             }
         }
 
+        internal string LogOutputProperties{ get;  set; }
```

Comment:
I think I'd like to see this called something like `DistinguishingProperties`?

Reply:
This property was removed and changed to a parameter that is passed. So this should be good now